app/src/sharedTest/java/org/oppia/app/utility/ClickActions.kt

```diff
@@ -6,7 +6,12 @@ import androidx.test.espresso.action.GeneralClickAction
 import androidx.test.espresso.action.Press
 import androidx.test.espresso.action.Tap
 
-fun clickAtXY(pctX: Float, pctY: Float): ViewAction {
+/**
+ * A custom [ViewAction] that can be used to perform click at a particular position.
```

Comment:
You should also mention that pointX and pointY are relative to the view that this action is being taken on, and are normalized from 0 to 1. Someone may try to use absolute pixel coordinates here and will be confused when it doesn't work.

Reply:
done